test/e2e/authorizer/authorizer_test.go

```diff
@@ -56,45 +56,49 @@ func newUserClient(t *testing.T, username, clusterName string, cfg *rest.Config)
 	cfgCopy.BearerToken = username + "-token"
 	kubeClient, err := kubernetes.NewForConfig(cfgCopy)
 	require.NoError(t, err)
-	kcpClient, err := kcp.NewForConfig(cfgCopy)
+	kcpClusterClient, err := kcp.NewForConfig(cfgCopy)
```

Comment:
hmm, this wrong now. NewClusterForConfig gives you a cluster client.

Reply:
Should be better now.